netbox/utilities/templatetags/helpers.py

```diff
@@ -389,6 +389,9 @@ def utilization_graph(utilization, warning_threshold=75, danger_threshold=90):
     """
     Display a horizontal bar graph indicating a percentage of utilization.
     """
+    is_full_danger = False
+    if utilization == 100:
+        is_full_danger = True
```

Comment:
This logic can be cleaned up considerably. There should be no need to store a variable indicating whether the utilization value is 100: we can just reference utilization directly.

Reply:
yeah, i misunderstand